recipes-temporary-patches/boot2qt/connman_%.bbappend

```diff
@@ -0,0 +1,5 @@
+
+FILESEXTRAPATHS_prepend := "${BSPDIR}/sources/meta-boot2qt/meta-boot2qt-distro/recipes-connectivity/connman/files:"
```

Comment:
This doesn't seem right. The "sources" directory may or may not exist, so hard-coding it here doesn't seem to be a good idea.

Reply:
Though I am not sure since I see such usage in many places in meta-pelux as well as other layers

Reply:
Does it work, if we do something like this?
${LAYERDIR}/../meta-boot2qt/meta-boot2qt-distro/recipes-connectivity/connman/files:

Reply:
just copy the patch to the same dir with bbappend